### ISSUE TITLE
not working L & cm eleven app

### ISSUE THREAD

**Issue by eurochild-dps** [ultramancool/AudioPrivacy#8]
album art is not disabled using latest xposed fw, cm12 or other rom that have the latest eleven music app.
+ a nice addition would be the option to disable the music visualizer in lockscreen

![screenshot_2015-03-09-18-15-56](https://cloud.githubusercontent.com/assets/7645326/6558996/6fcc83ba-c688-11e4-8676-a4d49df8db92.png)

**Comment by ultramancool** [comment]
I have no idea what's adding that visualizer to the lock screen, but I'm guessing that is your issue. It appears to work fine on stock L.

**Comment by eurochild-dps** [comment]
eleven (cm's music app) is adding the visualizer, the thing is not the visualizer but the album art not getting disabled in cm and propably other that stock roms and this music app

**Comment by Arcaena** [comment]
There's an option for disabling the visualizer in cm12 now, and I've opened an issue on their jira about getting an option to disable album art, but it's not looking promising. 

tl;dr cm12 album art still shows

**Comment by eurochild-dps** [comment]
yes i know and now they have an option to disable album art too, thanks for your time!

**Comment by ultramancool** [comment]
Please try new release compatible with 5.1.
